File: PHP/LED_Numbers.php
<?php
	$number = $argv[1];
	$leds = array(array(' _  ','| | ','|_| '),
				  array('  ','| ','| '),
				  array(' _  ',' _| ','|_  '),
				  array('_  ','_| ','_| '),
				  array('    ','|_| ','  | '),
				  array(' _  ','|_  ',' _| '),
				  array(' _  ','|_  ','|_| '),
				  array('_   ',' |  ',' |  '),
				  array(' _  ','|_| ','|_| '),
				  array(' _  ','|_| ',' _| '));
	for($i = 0; $i < 3; $i++){
		for($j = 0; $j < strlen($number); $j++){
			echo $leds[$number[$j]][$i];
		}
		echo "\n";
	}
?>

File: Kotlin/LEDNumbers.kt
package LEDNumbers

fun main(args: Array<String>) {
	if (args.size == 0) {
      println("Please provide a number...")
      return
    }
    val NumList:List<String> = args[0].split("")
	val Leds = mapOf("0" to listOf(" _  ", "| | ", "|_| "), 
	                 "1" to listOf("  ", "| ", "| "),
	                 "2" to listOf(" _  "," _| ","|_  "),
	                 "3" to listOf("_  ","_| ","_| "),
	                 "4" to listOf("    ","|_| ","  | "),
	                 "5" to listOf(" _  ","|_  "," _| "),
	                 "6" to listOf(" _  ","|_  ","|_| "),
	                 "7" to listOf("_   "," |  "," |  "),
	                 "8" to listOf(" _  ","|_| ","|_| "),
	                 "9" to listOf(" _  ","|_| "," _| "))
	for(i in 0..2){
		for(j in 1..NumList.size - 2){
			print(Leds[NumList[j]]!![i])
		}
		print("\n")
	}                 
}
